.github/workflows/release.yml

```diff
@@ -35,7 +35,7 @@ jobs:
             os: macos-latest
             target: aarch64-apple-darwin
 
-          - name: macos-latest
+          - name: x86_64-apple-darwin
```

Comment:
We talked about this briefly in sync this morning.. maybe it doesn't belong in this PR, but hopefully it doesn't conflict.